docs/content/latest/yugabyte-platform/administer-yugabyte-platform/back-up-restore-k8s.md

```diff
@@ -0,0 +1,129 @@
+
+    <br/>
+
+    * `k8s_namespace` specifies the kubernetes namespace where the Yugabyte Platform pod is running.
+    * `k8s_pod` specifies the name of the Yugabyte Platform kubernetes pod.
+
+1. Run the `yb_platform_backup.sh` script using the `backup` command:
+
+    ```sh
+    ./yb_platform_backup.sh backup --output <output_path> --k8s_namespace <kubernetes_namespace> --k8s_pod <kubernetes_pod> [--exclude_releases --verbose]
+    ```
+
+    * `backup` is the command to run the backup of the Yugabyte Platform server.
+    * `output_path` specifies the location for the output backup archive.
+    * `k8s_namespace` specifies the Kubernetes namespace in which the Yugabyte Platform pod is running.
+    * `k8s_pod` specifies the name of the Yugabyte Platform Kubernetes pod.
+    * `exclude_releases` excludes Yugabyte releases from the backup archive.
+    * `verbose` prints debug output.
+
+    {{< note title="Prometheus metrics data" >}}
+
+Backup and restore of Prometheus metrics data is not currently supported.
+
+    {{< /note >}}
+
+1. Verify that the backup `.tar` file, with the correct timestamp, is in the specified output directory.
+
+1. Upload the backup file to your preferred storage location, and delete it from the local disk.
+
+## Restore a Yugabyte Platform server
+
+To restore the Yugabyte Platform content from your saved backup, do the following:
+
+1. Copy the the Yugabyte Platform backup script (`yb_platform_backup.sh`) from the yugabyte-db repository to your local workstation using the following wget command:
+
+    ```sh
+    wget https://github.com/yugabyte/yugabyte-db/blob/master/managed/devops/bin/yb_platform_backup.sh
+    ```
+
+1. Verify that the machine performing the backup operation can access the Yugabyte Platform kubernetes pod instance:
+
+    ```sh
+    kubectl exec --namespace <kubernetes_namespace> -it <kubernetes_pod> -c yugaware -- cat /opt/yugabyte/yugaware/README.md
+    ```
+
+    <br/>
+
+    The Yugabyte Platform console is used in a highly available mode and orchestrates and manages YugabyteDB universes, or clusters, on one or more regions (across public cloud and private on-premises data centers). For more details, refer to the [Yugabyte Platform overview](https://docs.yugabyte.com/latest/yugabyte-platform/overview/).
+
+    <br/>
+
+    * `k8s_namespace` specifies the kubernetes namespace where the Yugabyte Platform pod is running.
+    * `k8s_pod` specifies the name of the Yugabyte Platform kubernetes pod.
+
+1. Run the `yb_platform_backup.sh` script using the `restore` command:
+
+    ```sh
+    ./yb_platform_backup.sh restore --input <input_path> --k8s_namespace <kubernetes_namespace> --k8s_pod <kubernetes_pod> [--verbose]
+    ```
+
+    <br/>
+
+    * `restore` is the command to restore the Yugabyte Platform server content.
+    * `input_path` is the path to the .tar backup file to restore.
```

Comment:
Reword (.tar backup file -> `.tar.gz` backup file):
* `input_path` is the path to the `.tar.gz` backup file to restore.